LICENSE-CONTRIBUTOR/ccla-simulalabs.txt

```diff
@@ -25,7 +25,7 @@ digitally sign the committed file, otherwise send a signed Agreement
 to MortBay.
 
 Each developer covered by this agreement should have their name appended 
-the Schedule A and the copy commited to LICENSES/ccla-CORPORATE-NAME.txt 
```

Comment:
I'm a little hesitant to change contents of LICENSE files.

Reply:
I understand. I will revert the changes on LICENSE files.